### PR TITLE
List options in `.stats` argument descriptions

### LINKED ISSUE #1329: List options in `.stats` argument descriptions
Currently the `.stats` argument description advises users to run the `get_stats()` function to see a list of available options. The list of statistics should instead be printed directly in the documentation for better readability.

### PR DESCRIPTION
**Pull request by edelarua**
# Pull Request

Fixes #1329

### CODE REVIEW

**Review by shajoezhu — APPROVED** [review]
Thanks so much @edelarua ! really like the changes! I was wondering if you could do a test on scda.test. once nothing breaks, we can merge it

**Comment by shajoezhu** [comment]
merge this if https://github.com/insightsengineering/scda.test/pull/156 is green

**Comment by shajoezhu** [comment]
https://github.com/insightsengineering/scda.test/actions/runs/11416119342/job/31766967284?pr=156#step:17:113 good to go

File: R/abnormal_by_marked.R [hunk 1]
```diff
@@ -28,8 +28,9 @@
 #' @inheritParams argument_convention
 #' @param category (`list`)\cr a list with different marked category names for single
 #'   and last or replicated.
-#' @param .stats (`character`)\cr statistics to select for the table. Run `get_stats("abnormal_by_marked")`
-#'   to see available statistics for this function.
+#' @param .stats (`character`)\cr statistics to select for the table.
+#'
+#'   Options are: ``r shQuote(get_stats("abnormal_by_marked"))``
```
Comment (Melkiades) [inline R/abnormal_by_marked.R:33]: maybe better than writing them all as we were thinking? wdyt @edelarua?

Comment (edelarua) [reply R/abnormal_by_marked.R:33]: I think it's nicer to keep it all together using `get_stats()` to keep things simple, we could make `get_stats()` internal if you think there's no benefit for users.

Comment (Melkiades) [reply R/abnormal_by_marked.R:33]: I see! lets keep get_stats then ;)

Comment (Melkiades) [reply R/abnormal_by_marked.R:33]: Probably we need to list all options in that documentation though

**Pull request merged**